code/modules/reagents/chemistry/holder.dm

```diff
@@ -53,20 +53,22 @@
 				GLOB.chemical_reactions_list[id] += D
 				break // Don't bother adding ourselves to other reagent ids, it is redundant.
 
-/datum/reagents/proc/remove_any(amount=1)
+/datum/reagents/proc/remove_any(amount = 1)
+	var/list/cached_reagents = reagent_list
```

Comment:
TG just uses this for every single time they utilize `reagent_list`. I don't like this; there's no reason to do this unless you're access the same list multiple times--and Id rather know, at a glance, if the list is being used, multiple times, in the same proc.